build.gradle.kts

```diff
@@ -28,6 +25,14 @@ kotlin {
     linuxArm64()
     mingwX64()
 
+    androidTarget {
+        publishLibraryVariants("release")
```

Comment:
By the way, the reason for adding this line is [a bug from `gradle-maven-publish-plugin`](https://github.com/vanniktech/gradle-maven-publish-plugin/issues/705)